packages/safety-suite-api/src/types.ts

```diff
@@ -56,6 +57,26 @@ export interface ApiResponse<T> {
   response: T;
 }
 
+export interface Page {
+  size: number;
+  totalElements: number;
+  totalPages: number;
+  number: number;
+}
+
+export type LinkName = 'first' | 'prev' | 'self' | 'next' | 'last';
+export interface Link {
+  href: string;
+}
+
+export type Links = {[R in LinkName]: Link};
```

Comment:
Not all responses will have all links.  Eg, if you are on the last page, there is no `next` response.  I think the correct syntax is `{[R in LinkName]?: Link};` but its been a hot sec since I've done Typescript and its friday afternoon.

Reply:
This worked.